core/src/components/modal/modal.tsx

```diff
@@ -756,7 +771,7 @@ export class Modal implements ComponentInterface, OverlayInterface {
           part="content"
           ref={el => this.wrapperEl = el}
         >
-          {showHandle && <div class="modal-handle" part="handle"></div>}
+          {showHandle && <button class="modal-handle" part="handle" type="button" onClick={this.onHandleClick}></button>}
```

Comment:
```suggestion
          {showHandle && <button class="modal-handle" part="handle" onClick={this.onHandleClick}></button>}
```

The type is redundant when we are using `<button>`